src/material-experimental/mdc-button/_button-base.scss

```diff
@@ -13,3 +13,12 @@
     @content;
   }
 }
+
+// MDC's disabled buttons define a default cursor with pointer-events none. However, they select
+// :disabled for this, which does not affect anchor tags.
+@mixin _mat-button-disabled() {
+  &[disabled] {
+    cursor: default;
+    pointer-events: none;
```

Comment:
This will cause the user to click on whatever is behind the button. Couldn't we do it with JS like we do for `MatAnchor` at the moment?

Reply:
Add a TODO?